addons/tm_tmf_loadouts/CfgLoadouts.hpp

```diff
@@ -172,6 +190,12 @@ class CfgLoadouts
 		category = "Russia";
 		#include "loadouts\ru_msv_emr.hpp"
 	};
+	class RUS_MSV_EMR_RATNIK
+	{
+		displayName = "RU Ground Forces (2018, EMR Ratnik)";
```

Comment:
You can specify that the loadout uses AK-12s, e.g. `(2018, EMR, AK-12s)`. You don't have to, but I like to do it for loadouts with non-standard guns